tests/test_lists2safebrowsing.py

```diff
@@ -168,3 +215,279 @@ def test_add_domain_to_list_duplicate():
+
+    assert output_writes == expected_output_writes
+    assert log_writes == expected_log_writes
+    assert capsys.readouterr().out == expected_print
+
+
+@pytest.mark.parametrize("list_type", [LIST_TYPE_ENTITY, LIST_TYPE_PLUGIN])
+def test_process_list_no_log_file(capsys, chunknum, list_type):
+    """Test entity/plugin list generation without a log file."""
+    (output_writes, _, expected_output_writes, _, expected_print) = \
+        _process_entity_or_plugin_list(chunknum, list_type, log=False)
+
+    assert output_writes == expected_output_writes
+    assert capsys.readouterr().out == expected_print
+
+
+def _get_entity_or_plugin_lists(list_type, config, section, chunknum,
+                                data, domains):
+    """Auxiliary function for get_entity_lists/get_plugin_lists tests."""
+    if list_type == LIST_TYPE_ENTITY:
+        function = l2s.get_entity_lists
+        url = ENTITY_LIST_URL
+    elif list_type == LIST_TYPE_PLUGIN:
+        function = l2s.get_plugin_lists
+        url = PLUGIN_LIST_URL
+    else:
+        raise ValueError("Invalid list type: '%s'" % list_type)
+
+    with patch("lists2safebrowsing.urllib2.urlopen",
+               mock_open(read_data=data)) as mocked_urlopen, \
+            patch("lists2safebrowsing.open", mock_open()) as mocked_open:
+        output_file, _ = function(config, section, chunknum)
+
+    urlopen_calls = mocked_urlopen.call_args_list
+    expected_urlopen_calls = [call(url)]
+    open_calls = mocked_open.call_args_list
+    output_filename = config.get(section, "output")
+    expected_open_calls = [call(output_filename, "wb"),
+                           call(output_filename + ".log", "w")]
+
+    # Exclude log writes
+    output_writes = output_file.write.call_args_list[len(domains):]
+    expected_output_writes = _get_expected_results(list_type, domains,
+                                                   chunknum, section)[0]
+
+    return (urlopen_calls, expected_urlopen_calls, open_calls,
+            expected_open_calls, output_writes, expected_output_writes)
+
+
+@pytest.mark.parametrize(
+    "section,version,testcase",
+    [pytest.param(section, version, id, id=id)
+        for id, section, version in GET_ENTITY_LISTS_TESTCASES]
+)
+def test_get_entity_lists(chunknum, section, version, testcase):
+    """Validate creating an entity list from a configuration section."""
+    config = ConfigParser.ConfigParser()
+    config.readfp(open("sample_shavar_list_creation.ini"))
+
+    if version:
+        config.set(section, "version", version)
+
+    data = json.dumps(TEST_ENTITY_DICT)
+    domains = TEST_ENTITY_LIST
+    if testcase == "separation_standard":
+        domains = [d for d in domains if d[0] != "Google"]
+    elif testcase == "separation_google":
+        domains = [d for d in domains if d[0] == "Google"]
+
+    (urlopen_calls, expected_urlopen_calls, open_calls,
+     expected_open_calls, output_writes, expected_output_writes) = \
```

Comment:
I recommend using `()` to the other places you are using `\` for line continuation as mentioned in https://www.python.org/dev/peps/pep-0008/#maximum-line-length